mu4e/mu4e-compose.el

```diff
@@ -331,17 +331,15 @@ trash folder, or nil if the message should be removed after
 sending."
   (when-let ((sent-dir
               ;; We can't call `functionp' since at least 'delete is a function.
-              (pcase mu4e-sent-messages-behavior
+              (pcase-exhaustive mu4e-sent-messages-behavior
                 ('delete nil)
                 ('trash (mu4e-get-trash-folder parent))
                 ('sent (mu4e-get-sent-folder parent))
                 ((pred functionp)
-                 (pcase (funcall mu4e-sent-messages-behavior)
+                 (pcase-exhaustive (funcall mu4e-sent-messages-behavior)
                    ('delete nil)
```

Comment:
No biggie, but In this case, I suppose it's nice to give a  mu4e-specific error. It's more informative than `pcase-exhaustive`...